README.md

```diff
@@ -25,6 +25,9 @@ defp deps do
 end
 ```
 
+> Note: If you use Arch along Phoenix framework then Phoenix `"=< 1.2"` uses `{:poison, "~> 2.0"}`. Therefore you may want to specify `"poison: "~> 2.1"`
```

Comment:
Arch should be Arc. Otherwise, thanks for opening this PR as I ran into this specific issue. 👍